### DISCUSSION TITLE
Is the project still alive?

### DISCUSSION THREAD

**Discussion by ennovative-pl** [react-page/react-page#1304] [General]
Last version is several months old, only 4 open issues, last discussion is the one that I've started last year... Is this project still actively maintained?

**Comment by macrozone** [comment]
yes it is, but there is a bit of summer break

**Comment by fschoning** [comment]
Is the summer break over yet, or has the project moved on? Thanks for letting us know the status.

**Reply by macrozone** [reply]
currently not alive unfortunatly, because I don't have a project where I can use it.

**Comment by fschoning** [comment]
Thanks for the update Marco.

Are you aware of any other open source projects that implement a web page editor component?

**Reply by macrozone** [reply]
I used [plasmic](https://www.plasmic.app/) in one project, which does a bit more then react-page, but its a paid service. But you can integrate it in you react or nextjs app and you can bring-your-own components like in react-page. I found it really developer friendly once you get the hang of it and you could do way more then with react-page. You can also create components directly in plasmic and even attach data sources like graphql or other apis to it. You can mix-and-match your own code components with components in plasmic as you need.

It is actually open-source, but i am not sure how easy it is to operate it yourself.

I decided in that project against react-page, because I also would have had to setup the whole page structure and a headless-cms for it, which is already done in plasmic (the content is served from plasmic)

https://www.builder.io/ also does a similar thing and is probably the most mature, but I did not yet had the chance to try it out.
